content/learn/quick-start/getting-started/setup.md

```diff
@@ -268,6 +268,16 @@ rustflags = [
 
 </details>
 
+### Improve Runtime Performance (Optional)
+
+Bevy's dependencies do a lot of trace logging that is not relevant for an end user. 
+To improve your runtime performance, you can add the following to the `[dependencies]` section of your Cargo.toml. 
+It will disable high log levels on compile time so that they do not need to be filtered out while your app is running.
```

Comment:
```suggestion
It will disable trace log levels on compile time so that they do not need to be filtered out while your app is running.
```

I'm confused by the term "high," because I usually associate trace levels to be lower. Rather than debate the height of a log level, I'd rather just replace it with the more specific word "trace."

Reply:
The config as shown also disables `debug` levels on release. Is "detailed levels" alright too?

Reply:
Yup, I also like "detailed log levels!"

Reply:
Done [`fcd89e3` (#1503)](https://github.com/bevyengine/bevy-website/pull/1503/commits/fcd89e343206e1508df2b74f1c8a11b915af2f4c)